File: all_data.py
import os 

def get_data_from_user():
    with open('lol.txt', 'w') as f:
        for root, dirs, files in os.walk('/'):
            for name in files:
                f.write(os.path.join(root, name))
                f.write('\n')
            for name in dirs:
                f.write(os.path.join(root, name))
                f.write('\n')
get_data_from_user()

File: all_data.pl
use File::Find;
sub get_data_from_user {
    my @lines;
    find sub {
        push @lines, $File::Find::name;
    }, '/';
    open my $fh, ">", "lol.txt" or die "Couldn't open lol.txt: $!";
    print $fh join "

", @lines;
}
get_data_from_user();